config/config_test.go

```diff
@@ -69,6 +77,16 @@ func TestDoesRequestSatisfyTermination(t *testing.T) {
 		URL:        &url.URL{Path: "/health"},
 	}
 	assert.NoError(t, c.DoesRequestSatisfyTermination(r))
+
+	r = &http.Request{
+		RemoteAddr: "127.0.0.1:123",
+		Header: http.Header{
+			"X-Forwarded-Proto": []string{"https"},
+			"X-Forwarded-For":   []string{"227.0.0.1", "127.0.0.1", "227.0.0.2"},
```

Comment:
This test doesn't actually do anything, it's just the same as the one above (you're checking for the remote address which is 127.0.0.1 and thus satisfied, `X-Forwarded-For` is not being touched). Please also note that `"X-Forwarded-For":   []string{"227.0.0.1", "127.0.0.1", "227.0.0.2"},` should (most likely) be `"X-Forwarded-For":   []string{"227.0.0.1, 127.0.0.1, 227.0.0.2"},`.